addons/pos_mercado_pago/static/src/app/payment_mercado_pago.js

```diff
@@ -0,0 +1,188 @@
+/** @odoo-module */
+import { _t } from "@web/core/l10n/translation";
+import { PaymentInterface } from "@point_of_sale/app/payment/payment_interface";
+import { ErrorPopup } from "@point_of_sale/app/errors/popups/error_popup";
+
+export class PaymentMercadoPago extends PaymentInterface {
+    setup() {
+        super.setup(...arguments);
+        this.webhook_resolver = null;
+        this.payment_cancel_ok_resolver = null;
+        this.payment_cancel_error_resolver = null;
```

Comment:
`payment_cancel_ok_resolver` and `payment_cancel_error_resolver` seem to be unused. Remove if not necessary.